LondonBritishLibrary/orient/BLorient772.xml

```diff
@@ -124,13 +123,50 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                             <citedRange unit="page">220a</citedRange>
                                         </bibl>
                                     </listBibl>
-                                    
+
                                 </source>
                             </recordHist>
                         </adminInfo>
                     </additional>
+
+                    <msPart xml:id="p1">
+                    <msIdentifier></msIdentifier>
+                    <msContents>
+                      <msItem xml:id="p1_i1">
+                        <locus from="3r"></locus>
+                        <title type="complete" ref="LIT1107Ankari"></title>
+                        <textLang mainLang="gez"></textLang>
+                        <incipit><locus target="3r"></locus>አንክሪጦስ፡ በእንተ፡
+                        ርትዕት፡ ሃይማኖት። ቃል፡ ዘ<persName ref="PRS3828Epiphani">ኢጴፋኒስ፡</persName>
+                         ዘሀገረ፡ ቍስጥንጥያ፡ ዘቄጵሮስ፡ እንዘ፡ ብሔሩሰ፡ ኤሌቴሎጰሊስ፡ እንተ ፍልስጤኔ፡
+                         ዘበህየ፡ ኮነ፡ አበ፡ መነኮሳት።</incipit></msItem>
+
+                         <msItem xml:id="p1_i2">
+                           <locus from="47r"></locus>
+                         <title ref="LIT1299Dersan">The 12 homilies of
+                          <persName ref="PRS8788Severos">Severus, bishop of
+                         ʾEsmūnāin</persName></title>
+                       </msItem>
+
+                    </msContents>
+                    </msPart>
+
+<msPart xml:id="p2">
+<msIdentifier></msIdentifier>
+<msContents>
+<msItem xml:id="p2_i1">
+<note>A leaf from the beginning of a copy of <ref type="work" corresp="LIT1957Mashaf">Qālämǝnṭos</ref> has
+been prefixed to this volume, <locus target="#2">f. 2</locus>, on the recto of which it is witten ቀሌምንጦስ፡ ዘቅ፡ መ፡ ዓለም።</note>
+</msItem>
+</msContents>
+</msPart>
+
```

Comment:
The description should follow the normal pattern, with msTitle, and the encoding of addition as an OwnershipNote.
And this msPart should be the first one. Each msPart could also contain the number of leaves it contains.

Reply:
you should eancode the name of the owner, you can do it in extras